pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -1074,6 +1074,52 @@
         doc = "Specify whether Client certificates are required for TLS Reject.\n"
             + "the Connection if the Client Certificate is not trusted")
     private boolean tlsRequireTrustedClientCertOnConnect = false;
+    @FieldContext(
+            category = CATEGORY_TLS,
+            doc = "Print the entire trust chain for the client's certificate in\n"
+                + "the application logs when authenticating\n"
+                + "Default value is 0 (off)")
+    private boolean tlsLogEntireCertificateChain = false;
+    @FieldContext(
+            category = CATEGORY_TLS,
+            doc = "If set to a positive non-zero value, the broker will print a warning in\n"
+                + "the application logs and increment a Prometheus counter if the client's\n"
+                + "certificate is within this many milliseconds of expiration.\n"
+                + "Default value is 0 (off)")
+    private long tlsPrintWarnOnClientCertNearingExpirationMillis = 0;
```

Comment:
Please check the provided value is not a negative value.

Reply:
Added negative checks.